### PR TITLE
ci: Fix failing Docker release by removing arm/v6 and arm/v7 support

### LINKED ISSUE #8904: New docker images aren't being created on releases
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the [latest version of Parse Server](https://github.com/parse-community/parse-server/releases). 

### Issue Description

Docker images stopped deploying on releases for 3+ months https://hub.docker.com/r/parseplatform/parse-server/tags. Instead, the dock CI hangs.

I have a workaround for the issue, will submit shortly, as the problem consist of all a couple of items I've seen in the docker community. The parse-dashboard repo doesn't have this issue because GitHub actions in that repo is only building for the `linux/amd64` platform which doesn't require QEMU or buildx.

### Steps to reproduce

Attempt to do a new release.

### Actual Outcome

No new image is created

### Expected Outcome

A new image on docker hub

### Environment

Server
- Parse Server version: `6.4.0` and `alpha`
- Operating system: `Linux`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `local`

Database
- System (MongoDB or Postgres): `N/A`
- Database version: `N/A`
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): `N/A`

Client
- SDK (iOS, Android, JavaScript, PHP, Unity, etc): `N/A`
- SDK version: `N/A`

### Logs

See https://github.com/parse-community/parse-server/actions/runs/7616018905/job/20741904342

### PR DESCRIPTION
**Pull request by cbaker6**
## Pull Request

- Report security issues [confidentially](https://github.com/parse-community/parse-server/security/policy).
- Any contribution is under this [license](https://github.com/parse-community/parse-server/blob/alpha/LICENSE).
- Link this pull request to an [issue](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

## Issue

The `arm/v6` and `arm/v7` aren't building in the docker CI.

Closes: #8904 

## Approach

Remove the `arm/v6` and `arm/v7` platform. These are Raspberry Pi 2 and 3 platforms. They can be added back in the future with another PR if someone wants to get them working.

### CODE REVIEW

File: .github/workflows/release-automated.yml [hunk 1]
```diff
@@ -78,7 +78,7 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           context: .
-          platforms: linux/amd64, linux/arm/v6, linux/arm/v7, linux/arm64/v8
```
Comment (cbaker6) [inline .github/workflows/release-automated.yml:81]: @mtrezza building for the `arm/v7` and sometimes `arm/v6` platforms was causing the issue.

**Comment by cbaker6** [comment]
@mtrezza this is ready for review. You can see an example working from my repo: https://hub.docker.com/layers/netreconlab/parse-server/7.0.0-alpha.23/images/sha256-47bb60954a7dc2431d38c45de6c8a0c2d6d058efe04fe4f523034e527b8c3242?context=explore

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ############################################################
 # Build stage
 ############################################################
-FROM node:18-alpine AS build
+FROM node:lts-alpine AS build
```
Comment (cbaker6) [inline Dockerfile:4]: Can switch back to `its` as `linux/amd64` and `linux/arm64/v8` seem to not have networking issues during the QEMU build

**Review by mtrezza — APPROVED** [review]
Let's give it a try...

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.0.0-alpha.24](https://github.com/parse-community/parse-server/releases/tag/7.0.0-alpha.24)

**Comment by cbaker6** [comment]
Seems to be working https://hub.docker.com/layers/parseplatform/parse-server/7.0.0-alpha.24/images/sha256-907710bc989318f71aafb9f846506e3cba513ef75ff28febfc44cff321536eec?context=explore

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.0.0-beta.1](https://github.com/parse-community/parse-server/releases/tag/7.0.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [7.0.0](https://github.com/parse-community/parse-server/releases/tag/7.0.0)

**Pull request merged**